core/scripts/keystone/src/generate_local_ocr3_config.go

```diff
@@ -0,0 +1,119 @@
+package src
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	"github.com/smartcontractkit/chainlink/deployment/keystone"
+	ksdeploy "github.com/smartcontractkit/chainlink/deployment/keystone"
+)
+
+type generateLocalOCR3Config struct{}
+
+func NewGenerateLocalOCR3ConfigCommand() *generateLocalOCR3Config {
+	return &generateLocalOCR3Config{}
+}
+
+func (g *generateLocalOCR3Config) Name() string {
+	return "generate-local-ocr3-config"
+}
+
+func (g *generateLocalOCR3Config) Run(args []string) {
+	publicKeys := []byte(`[
```

Comment:
why are there hardcoded values?

Reply:
I guess I assumed that a local setup script would accept some auth params for a set of nodes, query each of the nodes for the necessary key values, then build the `pubKeys` slice with the fetched data.